### PR TITLE
[WIP] Decompose Bio.Align and Bio.Intervals

### PR DESCRIPTION
**Pull request by Ward9250**
This will decompose Bio.Align and Bio.Intervals now that BioAlignments.jl and GenomicFeatures have release versions.

### CODE REVIEW

**Comment by Ward9250** [comment]
This is now passing, also fixes #476.

**Comment by Ward9250** [comment]
I'll merge shortly if no objections.

File: src/Align.jl [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# Module for biological sequence alignment.
+#
+# This file is a part of BioJulia.
+# License is MIT: https://github.com/BioJulia/Bio.jl/blob/master/LICENSE.md
+
+module Align
+
+using Reexport
+
+@reexport using BioAlignments
```
Comment (bicycle1885) [inline src/Align.jl]: Why not `const Align = BioAlignments`? I mean `Bio.Align` is an alias of `BioAlignments`.

Comment (Ward9250) [inline src/Align.jl]: Ok, I'll have to do that for the other modules too in that case, as up until now reexport has been used for them all.

Comment (Ward9250) [inline src/Align.jl]: Fixed :+1:

**Comment by bicycle1885** [comment]
BTW, Phylogenies.jl is a blocker to support Julia 0.6 on Bio.jl. Any plan to support Julia 0.6?

**Comment by Ward9250** [comment]
Absolutely, I will update to support 0.6

**Pull request merged**